Open Judge System/Services/OJS.Services/Business/ParticipantScores/ParticipantScoresBusinessService.cs

```diff
@@ -54,13 +54,8 @@ public void RecalculateForParticipantByProblem(int participantId, int problemId)
 
         public void NormalizeAllPointsThatExceedAllowedLimit()
         {
-            using (var scope = TransactionsHelper.CreateLongRunningTransactionScope())
-            {
                 this.NormalizeSubmissionPoints();
                 this.NormalizeParticipantScorePoints();
```

Comment:
These operations should be in one transaction

Reply:
After the last discussion, it was decided to remove the transactions. and submissions and Participant scores updates to be done separately